### ISSUE TITLE
combined uid fields for customTypes to support i18n

### ISSUE THREAD

**Issue by tilman** [boazpoolman/strapi-plugin-config-sync#39]
### Summary

Allow an array of keys for customTypes.uid config field and create a combined key with it.

### Why is it needed?

If i18n is enabled, Strapi will store two entries for each type instance. Right now the `customTypes.uid` field only accepts one key. It would be great to be able to set an array of keys to be able to also store multiple documents for customTypes which have i18n enabled.

### Suggested solution(s)

```
customTypes: [
                {
                    configName: 'product',
                    queryString: 'api::product.product',
                    uid: ['productId', 'locale'],
                }]
```

**Comment by boazpoolman** [comment]
Even better! Have at it :)

**Comment by boazpoolman** [comment]
@tilman if you have any questions or need help with this don’t hesitate to contact me 🙃

**Comment by tilman** [comment]
Sure! I think I'm nearly done. Will open a new PR in a few minutes so we can discuss on the changes :)

**Comment by boazpoolman** [comment]
Released with `v1.0.0-beta.5`
